README.md

```diff
@@ -130,18 +130,15 @@ commands and ``help <command>`` to print details on the specified command:
 
 	yosys> help help
 
-reading the design using the Verilog frontend:
+reading and elaborating the design using the Verilog frontend:
 
-	yosys> read_verilog tests/simple/fiedler-cooley.v
+	yosys> read -sv tests/simple/fiedler-cooley.v
```

Comment:
Is `-sv` necessary?

Reply:
You can also use `-vlog95` or `-vlog2k` or `-sv2005` etc, but `read` does expect an option indicating the file type as first argument.